packages/@aws-cdk/aws-route53/README.md

```diff
@@ -28,6 +28,25 @@ const zone = new route53.PrivateHostedZone(this, 'HostedZone', {
 
 Additional VPCs can be added with `zone.addVPC()`.
 
+### Adding Alias
+
+To add an alias to your zone:
+```ts
+import route53 = require('@aws-cdk/aws-route53');
+
+// For the example, using existing hosted zone
+const _baz = route53.HostedZone.import(this, 'baz', {
```

Comment:
I don't think `_baz` is consistent with our naming conventions. Change to `baz`?

Reply:
will change.